schema/codelists/itemClassificationScheme.csv

```diff
@@ -7,3 +7,6 @@ CPC,Central Product Classification,"The Central Product Classification (CPC) is
 OKDP,Russian National Classification of Products by Economic Activity (1994),"Part of the unified system of classification and coding of technical, economic and social information of the Russian Federation, in use for Russian Procurement information until January 1, 2016. Superseded by OKPD.",http://base.consultant.ru/cons/cgi/online.cgi?req=doc;base=LAW;n=143095;fld=134;from=14052-7;rnd=0.33374630799517035
 OKPD,Russian National Classification of Products by Economic Activity (2014),"Part of the national standardization system of the Russian Federation and based on harmonization with the Statistical Classification of Products by Activity in the European Economic Community, 2008. In use from January 1 2014.",http://base.consultant.ru/cons/cgi/online.cgi?req=doc;base=LAW;n=163703;fld=134;from=170864-15;rnd=0.39699640963226557
 CUCOP,Mexican Single Classifier of Public Procurement,"The Single Classifier of Public Procurement (Clasificador Único de las Contrataciones Públicas) is a national scheme in Mexico for the classification of goods, works, services and leases.",https://sites.google.com/site/cnetcucop/
+NAICS,North American Industry Classification System, "The North American Industry Classification System (NAICS) is the standard used by Federal statistical agencies in classifying business establishments for the purpose of collecting, analyzing, and publishing statistical data related to the U.S. business economy.",https://www.census.gov/eos/www/naics/downloadables/downloadables.html
+PSC,Product and Service Codes,"Also referred to as federal supply codes, product service codes are used by the United States government to describe the products, services, and research and development purchased by the government.",https://www.fpds.gov/downloads/service_product_codes.pdf
```

Comment:
I think https://www.acquisition.gov/PSC_Manual is a better link, as it has an Excel version. Where is the description text from?

Reply:
I get this when I access to that url
![image](https://user-images.githubusercontent.com/6834122/77648192-1dfd0c80-6f3e-11ea-97a7-3700fa156b63.png)
The description is from http://support.outreachsystems.com/resources/tables/pscs/

Reply:
The page loads for me. Can you check again?

Reply:
Nop, I still get the same

Reply:
I used my VPN to connect from Chile (they don't offer Paraguay), and I see that page now.

That URL is the only one with the most up-to-date PSC codes (March 2020). The current link is from 1998.

Since this is US-specific (and we have no US publishers), let's use the acquisition.gov URL for now as there is no alternative with up-to-date data.

Reply:
Sure, done!